test/unit/server/getters/get-winning-balance.js

```diff
@@ -1,11 +1,11 @@
 const { BigNumber } = require("bignumber.js");
-const setupTestDb = require("../../test.database");
+const { setupTestDb, seedDb } = require("../../test.database");
 const { dispatchJsonRpcRequest } = require("src/server/dispatch-json-rpc-request");
 
 describe("server/getters/get-winning-balance", () => {
   let db;
   beforeEach(async () => {
-    db = await setupTestDb();
+    db = await setupTestDb().then(seedDb);
```

Comment:
Is the `setupTestDb` ever run without the `seedDb` ? Wondering if it makes sense to do it in one place instead of in each getter? More a style thing than anything, the way you did it here might be more flexible

Reply:
I see, so as you migrate to this system, you drop the `.then(seedDb)`, then only provide the log ingestion?

Reply:
Yup!